SublimeLinter.sublime-settings

```diff
@@ -88,6 +88,16 @@
     // Highlight problems in the minimap.
     "show_marks_in_minimap": true,
 
+    // Display counters in the status bar.
+    // {warning} and {error} will be replaced with the actual numbers.
+    // Set to "" to display nothing
+    "statusbar.counters": "W: {warning} E: {error}",
```

Comment:
I prefer `counter_template`. At least something that contains "template".
Same for the messages setting.

I mean, you already use variables named like this, so why not expose this more descriptive name to the user?

Reply:
I opted for brevity, but you're right that's better.